server/controllers/assess/submittedApplicationsController.ts

```diff
@@ -58,6 +58,9 @@ export default class SubmittedApplicationsController {
     }
   }
 
+  /**
```

Comment:
as above, think this commit can be dropped?

Reply:
No changes are required to `submittedApplicationsController.ts` so these comments no longer exist.